src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -44,6 +50,80 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses a range in the format "a-b" into a list of {@code Index} objects.
+     *
+     * @param range The range string to parse.
+     * @return A list of {@code Index} objects from start to end of the range.
+     * @throws ParseException If the format is invalid or bounds are incorrect.
+     */
+    public static List<Index> parseInterval(String range) throws ParseException {
+        String[] bounds = range.split("-");
+        System.out.println(bounds.toString());
```

Comment:
Should probably remove the System.print statement. Can use logging instead if necessary

Reply:
Changes have been pushed